analytics_data_api/v0/views/learners.py

```diff
@@ -174,14 +174,21 @@ def get_queryset(self):
         """
         self._validate_query_params()
         query_params = self.request.QUERY_PARAMS
+
+        # Ordering by problem_attempts_per_completed can be ambiguous because
+        # it's a ratio values could be infinite (e.g. divide by zero) if no problems
+        # were completed.  Instead, sorting by attempt_ratio_order will produce
+        # a sensible ordering
+        order_by = query_params.get('order_by')
+        order_by = 'attempt_ratio_order' if order_by == 'problem_attempts_per_completed' else order_by
```

Comment:
If `'attempt_ratio_order'` is meant to be a secondary sort, I don't think this will work - this will order by `'attempt_ratio_order'` the primary sort field, and no secondary.

What could work would be to drop the logic into the `RosterEntry.get_users_in_course` method. If it's `order_by` parameter is `'problem_attempts_per_completed'`, you can construct the ES query such that we build the primary/secondary sorts appropriately.